elements/rh-blockquote/rh-blockquote.ts

```diff
@@ -0,0 +1,74 @@
+import { LitElement, html, css, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { pfelement } from '@patternfly/pfe-core/decorators.js';
+
+import styles from './rh-blockquote.css';
+
+/**
+ * A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @summary  A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @slot quote   - Provide a quote for the blockquote
+ * @slot author  - Provide an author for the blockquote
+ * @slot title   - Provide an author title for the blockquote
+ *
+ */
+@customElement('rh-blockquote') @pfelement()
```

Comment:
```suggestion
@customElement('rh-blockquote')
```

We don't need the `@pfelement` decorator, really